src/app/content/components/BookBanner.tsx

```diff
@@ -0,0 +1,128 @@
+import Color from 'color';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled, { css } from 'styled-components';
+import { ChevronLeft } from 'styled-icons/boxicons-regular/ChevronLeft';
+import { maxNavWidth } from '../../components/NavBar';
+import { h3MobileLineHeight, h3Style, h4Style } from '../../components/Typography';
+import theme from '../../theme';
+import { AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, Page } from '../types';
+import { bookDetailsUrl, findArchiveTreeSection } from '../utils';
+
+export const bookBannerDesktopHeight = 13;
+export const bookBannerMobileHeight = 10.4;
```

Comment:
How were these determined? Can these numbers be computed based on other numbers? If not, would a comment in the code help with maintenance?

Reply:
they're made up in the design, not sure what comment we should add?